lms/views/api/exceptions.py

```diff
@@ -155,14 +161,13 @@ def external_request_error(self):
         )
 
     @exception_view_config(context=OAuth2TokenError)
-    def oauth2_token_error(self):
-        return self.error_response()
+    def oauth2_token_error(self):  # pylint:disable=no-self-use
```

Comment:
The PyLint disable is annoying but its legit: the `oauth2_token_error()` view is in the view class because it inherits the class's `@view_defaults`, not because it uses `self`